src/ontology/mondo-ingest.Makefile

```diff
@@ -563,12 +563,18 @@ $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
 $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-content-robot-headers.json
 	mkdir -p $(EXTERNAL_CONTENT_DIR)
 	python ../scripts/add-robot-template-header.py $^ > $@
-.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
+.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv
 
-.PHONY: external-content-nord
+$(EXTERNAL_CONTENT_DIR)/nando-mappings.robot.tsv: $(MAPPINGSDIR)/nando-mondo.sssom.tsv
+	mkdir -p $(EXTERNAL_CONTENT_DIR)
+	# python $< $@ .... TODO @joeflack4 to fill in
```

Comment:
@joeflack4 this row is for you using your sssom -> mondo robot template converter

Reply:
@matentzn Roger that, I'll whip something up.

Reply:
@matentzn Ready for your review. I'll leave some more comments, but I'll drop the [robot template google sheet](https://docs.google.com/spreadsheets/d/1A4uJAOkFPnZ9VQj_tk8Hh1Auwdi9MuXtLflHGzW4LfM/edit#gid=581634025) here.

Reply:
@matentzn I assume this looks good? Can you check these boxes above and marked resolved if so?

Reply:
The output looks good!